### PR TITLE
Add rake task to run dependabot scan

### PR DESCRIPTION
**Pull request by jithindasad**
Fixes: #25

### CODE REVIEW

File: app/views/repositories/index.html.erb [hunk 1]
```diff
@@ -5,6 +5,8 @@
     Repositories of : <%= @current_user.fullname %>
   </h1>
   <p>Found <%= @repositories.length %> <span class="has-text-success">Public</span> Gitlab repositories</p>
+  <br>
+  <%= button_to "Refresh repositories", repositories_path %>
```
Comment (thegeorgeous) [inline app/views/repositories/index.html.erb:9]: Is this related to the Rake task?

**Comment by jithindasad** [comment]
@thegeorgeous will be cleaning up service class on next PR

**Review by thegeorgeous — APPROVED** [review]
